calculator.rb

```diff
@@ -0,0 +1,110 @@
+def print_operator_list(list)
+    count = 1
+    list.each do |key, value|
+        puts "#{count}. #{key}(#{value})"
+        count += 1
+    end
+end
```

Comment:
Perfect! I really like how you separated this functionality of printing the options with a method.